hilti/src/rt/library.cc

```diff
@@ -77,7 +77,7 @@ hilti::rt::Library::~Library() {
 }
 
 hilti::rt::Result<hilti::rt::library::Version> hilti::rt::Library::open() const {
-    constexpr auto mode = RTLD_LAZY | RTLD_GLOBAL;
+    constexpr auto mode = RTLD_NOW | RTLD_GLOBAL;
```

Comment:
Why this change? I could imagine `RTLD_NOW` to introduce issues for underlinked libraries. While we do not produce such libraries during JIT I am unsure we should disallow it here, at this low level.

EDIT: Does this change belong to the changes in dd571e95249d0dea6a1754bfa46983477caf3b27?

Reply:
The change isn't strictly necessary, but I ended up trying this 3 separate times while doing performance measurement to see if it could make a difference. It didn't, but just so that we don't keep wondering if it might, I left it at `NOW`.